bundles/org.openhab.binding.smgw/README.md

```diff
@@ -0,0 +1,36 @@
+# SMGW Binding
```

Comment:
For consistency, maybe rather "PPC SMGW Binding".

A full configuration example would have been appreciated but this can be added later.